internal/app/app.go

```diff
@@ -49,13 +50,16 @@ type DynamoClient interface {
 	Put(ctx context.Context, v interface{}) error
 	Create(ctx context.Context, v interface{}) error
 	DeleteKeys(ctx context.Context, keys []dynamo.Key) error
+	DeleteOne(ctx context.Context, key dynamo.Key) error
+	Update(ctx context.Context, input *dynamodb.UpdateItemInput) error
```

Comment:
I'm not sure about exposing the `dynamodb` type here, we avoid it in the other signatures. Would `Update(ctx context.Context, pk, sk string, values map[string]types.AttributeValue, expression string) error` be any nicer?

Reply:
That works for me - slightly less boilerplate code to be passing in.